$NetBSD: patch-configure,v 1.1 2018/11/01 21:12:40 adam Exp $

Use absolute paths for dlopen(); don't expect users set LD_LIBRARY_PATH.

--- configure.orig	2018-11-01 20:04:16.000000000 +0000
+++ configure
@@ -11150,7 +11150,7 @@ find_lib()
         host_lib_path="/usr/$base_libdir /usr/local/$base_libdir"
     fi
     for path in $env_lib_path $gcc_bin_path $gcc_lib_path $host_lib_path; do
-        lib=`ls -- $path/$1 2>/dev/null | sed -e '/\.so\..*\./d' -e 's,.*/,,' | sort | tail -1`
+        lib=`ls -- $path/$1 2>/dev/null | sed -e '/\.so\..*\./d' | sort | tail -1`
         if test x$lib != x; then
             echo $lib
             return
